client_cli/src/main.rs

```diff
@@ -1060,6 +1062,196 @@ mod peer {
     }
 }
 
+mod param {
+    use super::*;
+    use clap::Subcommand;
+    use core::num::NonZeroU64;
+    use iroha::client::{self, *};
+    use iroha_data_model::isi::SetParameter;
+    use iroha_data_model::parameter::{self, *};
+
+    // #[derive(Parser, Debug)]
+    // #[command(author, version, about, long_about = None)]
+    #[derive(Subcommand, Debug)]
+    pub enum Args {
+        // #[command(subcommand)]
+        // Parameters(ParamCmd),
+        #[command(subcommand)]
+        Get(Box<Get>),
+        #[command(subcommand)]
+        Set(Box<Set>),
+    }
+
+    impl RunArgs for Args {
+        fn run(self, context: &mut dyn RunContext) -> Result<()> {
+            match_all!((self, context), { Args::Get, Args::Set, })
+        }
+    }
+
+    #[derive(Subcommand, Debug)]
+    pub enum Get {
+        All,
+        #[command(flatten)]
+        Inner(ParamCmd),
+    }
+
+    /// Get values of parameters
+    impl RunArgs for Get {
+        fn run(self, context: &mut dyn RunContext) -> Result<()> {
+            let client = context.client_from_config();
+            let vec = match self {
+                Self::All => client
+                    .request(client::parameter::all())
+                    .wrap_err("Failed to get all the parameters."),
+                Self::Inner(param) => client
+                    .build_query(client::parameter::all())
+                    .with_filter(ParamCmd::Sumeragi)
+                    .execute()
+                    .wrap_err("Failed to get the parameter value"),
+            }?;
+            context.print_data(&vec.collect::<QueryResult<Vec<_>>>()?)?;
+            Ok(())
+        }
+    }
```

Comment:
So the first thing is here, I tried to replicate this from another commit, but I'm stuck on how to get the specific parameters that are in `ParamCmd`. I'd like some suggestion on that.

Reply:
You will also need to make a version of `ParamCmd` and its nested types without the `value` field for this.

Reply:
> You will also need to make a version of `ParamCmd` and its nested types without the `value` field for this.

@nxsaken I would need to duplicate it again? That seems weird. Maybe `ParamCmd` could be refactored for the better?